webapp/crashstats/libbugzilla.py

```diff
@@ -80,40 +106,55 @@ def crash_report_to_description(crash_report_url, processed_crash):
         lines.append("")
         lines.append(f"Reason: ```{processed_crash['reason']}```")
 
-    # If there's a java_stack_trace, add that; this is a big string blob, so we truncate
-    # it at 5,000 characters but don't otherwise format it
-    if processed_crash.get("java_stack_trace"):
+    frames = None
+    if "json_dump" in processed_crash:
+        # Generate frames from the stackwalker output from parsing a minidump
+        threads = processed_crash.get("json_dump", {}).get("threads")
+        if threads:
+            if processed_crash.get("crashing_thread") is None:
+                lines.append("")
+                lines.append("No crashing thread identified; using thread 0.")
+
+            thread_index = processed_crash.get("crashing_thread") or 0
+            frames = minidump_thread_to_frames(threads[thread_index])
+
+    if not frames and "java_exception" in processed_crash:
+        stacktraces = (
+            processed_crash["java_exception"].get("exception", {}).get("values", [])
+        )
```

Comment:
also a good place for `:=`
```suggestion
    if not frames and java_exception := processed_crash.get("java_exception"):
        stacktraces = java_exception.get("exception", {}).get("values", [])
```